phylogenetic/defaults/auspice_config.json

```diff
@@ -0,0 +1,88 @@
+{
+  "title": "Norovirus Build",
+  "maintainers": [
+    {"name": "the Nextstrain team", "url": "https://nextstrain.org/team"}
+  ],
+  "data_provenance": [
+    {
+      "name": "GenBank",
+      "url": "https://www.ncbi.nlm.nih.gov/genbank/"
+    }
+  ],
+  "build_url": "https://github.com/nextstrain/norovirus",
+  "colorings": [
+    {
+      "key": "ORF2_type",
+      "title": "Vp1 Genotype",
+      "type": "categorical"
+    },
+    {
+      "key": "ORF1_type",
+      "title": "Rdrp Genotype",
+      "type": "categorical"
+    },
+    {
+      "key": "num_date",
+      "title": "Date",
+      "type": "continuous"
+    },
+    {
+      "key": "country",
+      "title": "Country",
+      "type": "categorical"
+    }
+  ],
+  "geo_resolutions": [
+    "country",
+    "region"
+  ],
+  "panels": [
+     "tree",
+     "map",
+     "entropy",
+     "frequencies"
+  ],
+  "display_defaults": {
+    "map_triplicate": true,
+    "color_by":"ORF2_type"
```

Comment:
[caveat: I am not an expert on noro biology & clasification schemes]

For the non-all datasets I'd suggest `ORF1_type` (RdRp genotype) as the current defaults show no variation within each dataset - i.e. the tree is entirely one colour, and the frequencies panel is just one stream.